tests/manifests/test_manifest-empty.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from lib.manifest import Manifest
+
+manifest_data = {}
+manifest = Manifest(manifest_data, "empty-manifest")
+    
+def test_botname():
+    assert manifest.botname() == "GPT"
+
+def test_username():
+    assert manifest.username() == "You(empty-manifest)"
+
```

Comment:
```suggestion
import pytest
import sys

from lib.manifest import Manifest

@pytest.fixture
def manifest():
    manifest_data = {}
    return Manifest(manifest_data, "empty-manifest")
    
def test_botname(manifest):
    assert manifest.botname() == "GPT"

def test_username(manifest):
    assert manifest.username() == "You(empty-manifest)"

```

PRありがとうございます！何点か修正を提案させてください！

1. `python -m pytest` という形でテストを実行すれば `sys.path.append(os.path.join(os.path.dirname(__file__), '..'))` は不要です。
2. pytest では定数（今回のテストコードだと `manifest`）は Fixtures を利用するのが一般的です。

Reply:
test_manifest.py のほうも同様に修正してみてください！ 🙇